File: src/main/java/io/sch/historyscan/domain/contexts/analysis/clocrevisions/filesystem/CodebasePathCanNotBeRead.java
package io.sch.historyscan.domain.contexts.analysis.clocrevisions.filesystem;

import io.sch.historyscan.domain.error.HistoryScanTechnicalException;

import java.io.IOException;

public class CodebasePathCanNotBeRead extends HistoryScanTechnicalException {
    public CodebasePathCanNotBeRead(String message, IOException e) {
        super(message, e);

    }
}

File: historyscan-back/src/main/kotlin/io/sch/historyscan/domain/contexts/analysis/clocrevisions/filesystem/CodebasePathCanNotBeRead.kt
package io.sch.historyscan.domain.contexts.analysis.clocrevisions.filesystem

import io.sch.historyscan.domain.error.HistoryScanTechnicalException
import java.io.IOException

class CodebasePathCanNotBeRead(message: String, e: IOException) : HistoryScanTechnicalException(message, e)
